apps/site/lib/site_web/views/content_view.ex

```diff
@@ -8,7 +8,7 @@ defmodule SiteWeb.ContentView do
 
   alias Content.Field.{File, Image, Link}
   alias Content.Paragraph
-  alias Content.Paragraph.{Callout, ColumnMulti, FareCard}
+  alias Content.Paragraph.{Callout, ColumnMulti, DescriptionList, FareCard}
```

Comment:
Wrong Paragraph. This is supposed to be for `ContentList.t()` -- empty content lists are still appearing on the website.